src/graph/graphQLClient.ts

```diff
@@ -73,16 +108,25 @@ export async function callSaveFavorite(
  *
  * @param accessToken accessToken of the user
  * @param consumerKey consumerKey associated with the user
+ * @param headers any headers received by proxy is just pass through to web graphQL proxy.
  * @param variables input variables required for the query
  */
 export async function callSavedItems(
   accessToken: string,
   consumerKey: string,
+  headers: any,
   variables: GetSavedItemsQueryVariables
 ): Promise<GetSavedItemsQuery> {
-  const client = getClient(accessToken, consumerKey);
-  return client.request<GetSavedItemsQuery, GetSavedItemsQueryVariables>(
-    GetSavedItemsDocument,
-    variables
-  );
+  try {
+    const client = getClient(accessToken, consumerKey, headers);
+    return client.request<GetSavedItemsQuery, GetSavedItemsQueryVariables>(
+      GetSavedItemsDocument,
+      variables
+    );
+  } catch (e) {
+    const err = `fails at callSavedItems` + JSON.stringify(e);
+    Sentry.captureException(err);
+    console.error(err);
+    throw err;
```

Comment:
since the errors thrown by graphQLClient are not super obvious, i am going to add some log statements to track where it fails